e2e/features/home-office-exposure.feature

```diff
@@ -0,0 +1,127 @@
+
+    Then I should only see the `homeOffice_uploadBundle` case progress image
+
+    And I should see the text `Do this next`
+    And I should see the text `An appeal against the Home Office decision in this case has been submitted. You can view the appeal form in the documents tab`
+    And I should see the text `If you accept that there is a right of appeal you should upload the Home Office bundle`
+    And I should see the text `If you consider the appeal is not valid, you should contact the Tribunal and the appellant with your reasons`
+
+    And I should see the case details
+    And I should not see the hearing details
+
+    And I should not see the option `Send direction` for the `Next step` field
+    And I should not see the option `Change the direction due date` for the `Next step` field
+    And I should see the option `Upload Home Office bundle` for the `Next step` field
+    And I should not see the option `Submit your case` for the `Next step` field
+    And I should not see the option `Add case note` for the `Next step` field
+    And I should not see the option `Record an application` for the `Next step` field
+    And I should not see the option `Build your case` for the `Next step` field
+
+    When I click the `upload the Home Office bundle` link
+    Then I am on the `Upload Home Office bundle` page
+    And I click the `Cancel` link
+
+    When I click the `documents tab` link
+    Then I am on the `Documents` page
+
+    # HO APC:
+
+    Given I switch to be a `Home Office APC`
+    And I click the `Overview` tab
+
+    Then I should only see the `homeOffice_uploadBundle` case progress image
+
+    And I should see the text `Do this next`
+    And I should see the text `An appeal against the Home Office decision in this case has been submitted. You can view the appeal form in the documents tab`
+    And I should see the text `If you accept that there is a right of appeal you should upload the Home Office bundle`
+    And I should see the text `If you consider the appeal is not valid, you should contact the Tribunal and the appellant with your reasons`
+
+    And I should see the case details
+    And I should not see the hearing details
+
+    And I should not see the option `Send direction` for the `Next step` field
+    And I should not see the option `Change the direction due date` for the `Next step` field
+    And I should see the option `Upload Home Office bundle` for the `Next step` field
+    And I should not see the option `Submit your case` for the `Next step` field
+    And I should not see the option `Add case note` for the `Next step` field
+    And I should not see the option `Record an application` for the `Next step` field
+    And I should not see the option `Build your case` for the `Next step` field
+
+    When I click the `upload the Home Office bundle` link
+    Then I am on the `Upload Home Office bundle` page
+    #And I click the `Cancel` link
+    #And I upload Home Office bundle
```

Comment:
We need to remove this line because we are not uploading any document as part of this test anyway it is commented

Reply:
Removed